utils/png2cc.py

```diff
@@ -0,0 +1,119 @@
+import os
+import argparse
+from pathlib import Path
+import logging
+import subprocess
+import shutil
+from wand.image import Image
+
+logger = logging.getLogger('png2cc.py')
+
+
+def bin2cc(src_filename: Path, dst_filename: Path, var_name: str):
+    if not src_filename.exists():
+        logger.error('no {} found', src_filename.name)
+        return 1
+    with open(src_filename.resolve(),
+              "rb") as src_file, open(str(dst_filename), "w") as dst_file:
+        dst_file.write("const uint8_t {}[] = ".format(str(var_name)) + "{\n")
+        byte = src_file.read(1)
+        while byte:
+            dst_file.write("\t0x{},\n".format(byte.hex()))
+            byte = src_file.read(1)
+        dst_file.write("};\n")
+
+
+def bincc_folder(src_folder: Path, dst_folder: Path):
+    if not dst_folder.exists():
+        logger.error('no {} found', dst_folder.name)
+        return 1
+
+    for filename in os.listdir(src_folder.resolve()):
+        if filename.endswith(".png"):
```

Comment:
If I wanted to be an evil user, I'd either create a filename with uppercase `.PNG` or a directory with `.png` name. But as this is a development utility, it doesn't have to be perfect and bullet proof, so it can be left as it is, it's good enough ‒ saying it just so you are aware.